### PR TITLE
Amazon WorkSpaces client

### PR DESCRIPTION
**Pull request by jimmyjones2**
Client to access [Amazon WorkSpaces](https://aws.amazon.com/workspaces/)

Unofficial wrapper of deb package, which is noted in AppData.

### CODE REVIEW

File: com.amazon.Workspaces.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+              "type": "debian-repo",
+              "package-name": "workspacesclient",
+              "root": "https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu",
+              "dist": "bionic",
+              "component": "main"
+          }
+        },
+        {
+            "type": "file",
+            "path": "com.amazon.Workspaces.appdata.xml"
```
Comment (LeandroStanger) [inline com.amazon.Workspaces.json:60]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Comment (jimmyjones2) [reply com.amazon.Workspaces.json]: Thanks for reviewing. Have pushed a change to move to metainfo.

**Comment by LeandroStanger** [comment]
bot, build com.amazon.Workspaces

**Pull request closed without merging**